docs/includes/reactivestreams/rsoperators.adoc

```diff
@@ -20,6 +20,18 @@ ifndef::rootdir[:rootdir: {docdir}/../..]
 
 :feature-name: Reactive Streams
 
+== Contents
+
+- <<Overview, Overview>>
+- <<maven-coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+- <<Reference, Reference>>
+
+== Overview
+
+Helidon implements link:{microprofile-rs-operators-spec-url}[MicroProfile Reactive Streams Operators] specification which defines reactive operators and provides a standartized tool for manipulation with https://www.reactive-streams.org/[Reactive Streams]. Use it you want stick with MicroProfile ecosystem which guarantees source-level portability between different implementations of the specification.
```

Comment:
You can use the Helidon supported reactive operators and tooling when you want to maintain source-level portability between different implementations of the MicroProfile specifications.